Fix "C# custom telemetry section" anchor link

andynava opened the pull request:
Fix anchor link for "C# custom telemetry section" from "#create-custom-telemetry-data-in-c-function-code" to "#custom-telemetry-in-c-functions"

azurecla commented:
Hi __@arunvinoth__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

alancameronwills commented:
cool.

andynava committed:
Fix "C# custom telemetry section" anchor link

Fix anchor link for "C# custom telemetry section" from "#create-custom-telemetry-data-in-c-function-code" to "#custom-telemetry-in-c-functions"
